packages/react-aria-components/docs/FileTrigger.mdx

```diff
@@ -122,14 +122,35 @@ A file trigger can accept multiple files by passsing the `allowsMultiple` proper
 
 ## Directory selection
 
-To enable selecting directories instead of files, use the `directory` property. 
+To enable selecting directories instead of files, use the `acceptDirectory` property. 
 
 This reflects the [webkitdirectory](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/webkitdirectory) HTML attribute and allows users to select directories and their contents.
 
 ```tsx example
-<FileTrigger directory>
-  <Button>Select a directory</Button>
-</FileTrigger>
+function Example () {
+  let [files, setFiles] = React.useState([]);
+
+  return (
+    <>
+      <FileTrigger
+        acceptDirectory
+        allowsMultiple
+        onSelect={(e) => {
+          if (e) {
+            let fileList = [...e].map(file => file.webkitRelativePath);
```

Comment:
```suggestion
            let fileList = [...e].map(file => file.webkitRelativePath ?? file.name);
```

I tried in Safari and selected both a directory as well as a file from the same level and it was blank.
Also, I can select multiple directories in Safari